21/day01/src/lib.rs

```diff
@@ -0,0 +1,60 @@
+#![deny(missing_docs)]
+//! This crate provides a solution for the advent of code puzzle: 2021, day 1.
+
+fn parse_i32(s: &str) -> i32 {
+    s.parse::<i32>().unwrap()
+}
+
+fn add_three(a: &str, b: &str, c: &str) -> i32 {
+    parse_i32(a) + parse_i32(b) + parse_i32(c)
+}
+
+fn part1(input: &str) -> i32 {
+    let mut input_split_on_lines = input.split('\n');
```

Comment:
Instead of creating a variable **input_split_on_lines**, parse in one statement the value in an Integer-Type.